### PR TITLE
Mock Store

### PR DESCRIPTION
**Pull request by johnTurknett**
The initial attempt at a mock storefront.

### CODE REVIEW

**Comment by bakedbean** [comment]
Some thoughts re: naming conventions:
Generally when something is in a folder named "components", it's safe to assume everything in that folder and is a component.  Based on that assumption, we name components like "Component.js".  It seems redundant to be tagging `.component` onto every file.  I'd suggest adopting this convention so as to be in keeping with the Checkout/Apply UI we're developing.  So `checkout.component.js` would just become `Checkout.js`

File: src/pages/store/checkout/contact-information.page.js [hunk 1]
```diff
@@ -0,0 +1,479 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import CountrySelector from "../../../components/theme/checkout/country-selector.component";
+import StateSelector from "../../../components/theme/checkout/state-selector.component";
+import "./styles.css";
+import ck from "creditkey-js";
+import Checkout from "../../../components/checkout.component";
+
+function CheckoutPage() {
```
Comment (bakedbean) [inline src/pages/store/checkout/contact-information.page.js:9]: the state being set here should use the same dynamic values that are used in what is now the dev component.

**Comment by johnTurknett** [comment]
@bakedbean I will get that changed.

File: src/pages/store/cart.page.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+                <div className="cart-shipping">
+                  Shipping &amp; taxes calculated at checkout
+                </div>
+                <div className="cart-checkout">
+                  <Link
+                    to="/checkout/contact-information"
+                    className="button-primary mdc-ripple-surface mdc-ripple-upgraded"
+                  >
+                    <ShoppingCartIcon style={{ "margin-top": "5px" }} />
```
Comment (widox) [inline src/pages/store/cart.page.js:232]: think you want `marginTop` here

File: src/pages/store/checkout/contact-information.page.js [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+                </span>
+              </a>
+
+              <h1 className="visually-hidden">Information</h1>
+            </div>
+            <div className="sidebar__content">
+              <div
+                id="order-summary"
+                className="order-summary order-summary--is-collapsed"
+                style={{ 'margin-top': '35px' }}
```
Comment (widox) [inline src/pages/store/checkout/contact-information.page.js:302]: also `marginTop`

File: src/pages/store/checkout/contact-information.page.js [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+                                <span
+                                  className="product-thumbnail__quantity"
+                                  aria-hidden="true"
+                                >
+                                  1
+                                </span>
+                              </div>
+                            </td>
+                            <th className="product__description" scope="row">
+                              <span className="product__description__name order-summary__emphasis" style={{ 'margin-left': '10px'}}>
```
Comment (widox) [inline src/pages/store/checkout/contact-information.page.js:328]: and `marginLeft`

File: src/pages/store/product-show.page.js [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+                </div>
+              </div>
+
+              <br />
+
+              <div className="product-form--regular" data-product-form-regular="">
+                <div>
+                  <button
+                    className="product-form--atc-button mdc-ripple-surface mdc-ripple-upgraded"
+                    style={{ "margin-left": "-5px" }}
```
Comment (widox) [inline src/pages/store/product-show.page.js:88]: this `marginLeft` too 😃

**Comment by widox** [comment]
would it be possible to pull out the various hardcoded localhost URLs into `.env` or some common place instead?

File: src/pages/store/cart.page.js (hunk 2 above)
Comment (johnTurknett) [reply src/pages/store/cart.page.js:232]: Fixed.

File: src/pages/store/checkout/contact-information.page.js (hunk 3 above)
Comment (johnTurknett) [reply src/pages/store/checkout/contact-information.page.js:302]: Fixed.

File: src/pages/store/checkout/contact-information.page.js (hunk 4 above)
Comment (johnTurknett) [reply src/pages/store/checkout/contact-information.page.js:328]: Fixed.

File: src/pages/store/product-show.page.js (hunk 5 above)
Comment (johnTurknett) [reply src/pages/store/product-show.page.js:88]: Fixed.

**Comment by bakedbean** [comment]
can we always show the cart in the header, with zero items prior to adding something?

**Comment by johnTurknett** [comment]
> can we always show the cart in the header, with zero items prior to adding something?

Done and Done.

**Pull request merged**